### ISSUE TITLE
Canary 0.35.3-canary.89: Webpack Config error setting title on build

### ISSUE THREAD

**Issue by dthyresson** [redwoodjs/redwood#3215]
With `"@redwoodjs/core": "^0.35.3-canary.89"` when I `yarn rw build` the web side:

```terminal
[webpack-cli] Failed to load '/Users/x/Documents/projects/redwoodjs/redwoodblog-rbac/node_modules/@redwoodjs/core/config/webpack.production.js' config
[webpack-cli] TypeError: Cannot set property 'title' of undefined
    at /Users/x/Documents/projects/redwoodjs/redwoodblog-rbac/web/config/webpack.config.js:4:28
    at Array.forEach (<anonymous>)

```

The cause is the way I had set the title:

```js
module.exports = (config, { _env }) => {
  config.plugins.forEach((plugin) => {
    if (plugin.constructor.name === 'HtmlWebpackPlugin') {
      plugin.options.title = 'Redwood Blog with RBAC'
    }
  })

  return config
}
```

If I remove and use

```js
module.exports = (config, { _env }) => {
  return config
}
```

The my build succeeds.

If this is no longer the way (as SEO and MetaTags are now the better way) need to remove these docs:

https://redwoodjs.com/docs/webpack-configuration.html#changing-the-title-of-the-page

![image](https://user-images.githubusercontent.com/1051633/128547408-a3e71386-c8e0-48bf-81b1-f184a06d1f32.png)

**Comment by simoncrypta** [comment]
Oh yes I removed the title option in Webpack since Helmet is now set by default. I didn't notice this part of the doc, I can remove it in the PR I already open for the SEO doc update 🙂

**Comment by thedavidprice** [comment]
Fixed in #3026

Docs will be updated when v0.36 releases
